### PR TITLE
Use depkeeper-preset-yoshi

### PR DESCRIPTION
**Pull request by ihork**
### 🔦 Summary
Move depkeeper configuration to depkeeper-preset-yoshi. Left only logging functionality.

### 🗺️ Test Plan
Check only logging functionality

### CODE REVIEW

**Comment by ihork** [comment]
Will be available to merge only after publishing `depkeeper-preset-yoshi`. I'll keep you posted

**Comment by yairhaimo** [comment]
Any update on this?

**Comment by itaibenda** [comment]
@yairhaimo the package is published on `npmjs` under the @wix scope so it should be ready to merge

File: packages/yoshi/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "test:watch": "mocha './test/{,!(fixtures)/**/}/*.spec.js' --watch"
   },
   "dependencies": {
+    "@wix/depkeeper-preset-yoshi": "~1.0.0",
```
[the remark this answers is not in the document]
Comment (itaibenda) [reply packages/yoshi/package.json]: @yairhaimo you're absolutely right. Changed it to the latest minor.

**Comment by yairhaimo** [comment]
Thanks 🏆

**Pull request merged**